release.sh

```diff
@@ -3,39 +3,96 @@
 set -e
 set -x
 
-if [ ! -d ${HOME}/google-cloud-sdk ]; then
-  wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-206.0.0-linux-x86_64.tar.gz -O install.tar.gz
-  tar -xvzf install.tar.gz
+KEY_FILE=
+SPIN_GCS_BUCKET_PATH=
+VERSION=
+
+
+function process_args() {
+  while [[ $# > 0 ]]
+  do
+    local key="$1"
+    shift
+    case $key in
+      --gcs_bucket_path)
+        SPIN_GCS_BUCKET_PATH="$1"
+        shift
+        ;;
+      --key_file)
+        KEY_FILE="$1"
+        shift
+        ;;
+      --version)
+        VERSION="$1"
+        shift
+        ;;
+      --help|-help|-h)
+        print_usage
+        exit 13
```

Comment:
why 13? Doesnt -h usually succeed with 0?

Reply:
I took this from `InstallSpinnaker.sh`, which had it. Not sure either way. I can change this path to 0.